solver/src/driver.rs

```diff
@@ -309,6 +312,9 @@ impl Driver {
         futures::stream::iter(settlements)
             .filter_map(|settlement| async {
                 let surplus = settlement.settlement.total_surplus(prices);
+                // Given the fee discount the solver fees by themselves may not be sufficient to make a solution economically viable (leading to a negative objective value)
```

Comment:
I didn't understand the sentence. Did you mean this?

```suggestion
                // Given the fee discount the solver applies by themselves may not be sufficient to make a solution economically viable (leading to a negative objective value)
```